.github/test-spec.yml

```diff
@@ -126,7 +117,10 @@
   - "subsys/tracing/**/*"
 
 "CI-desktop-test":
-  - "**/*"
+  - any:
+      - "**/*"
+      - "!samples/bluetooth/**/*"
+      - "!tests/bluetooth/**/*"
```

Comment:
Maybe even `!samples/**/*` and `!tests/**/*`? These integration test suite is only for `applications/nrf_desktop` from nRF Connect SDK and changes to samples/tests should not affect it.